src/catalog/src/builtin.rs

```diff
@@ -2009,6 +2015,16 @@ pub static MZ_COMPUTE_OPERATOR_HYDRATION_STATUSES_PER_WORKER: Lazy<BuiltinSource
         access: vec![PUBLIC_SELECT],
     });
 
+pub static MZ_CATALOG_RAW: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSource {
+    name: "mz_catalog_raw",
+    schema: MZ_INTERNAL_SCHEMA,
+    oid: oid::SOURCE_CATALOG_RAW_OID,
+    data_source: BuiltinSourceType::Catalog,
+    desc: crate::durable::persist::desc(),
+    is_retained_metrics_object: false,
+    access: vec![MONITOR_SELECT],
```

Comment:
I know this is not ready for review, but I'm just leaving this comment so that I don't forget later. I think at first we'll probably want to restrict this to just `mz_system` and `mz_support`.